src/shared/books.ts

```diff
@@ -0,0 +1,5 @@
+export const getBooks = async () =>
+  new Promise<string[]>(resolve => {
+    const _getBooks = () => resolve(['Harry Potter 1', 'Harry Potter 2', 'Harry Potter 3']);
+    setInterval(_getBooks, 10000);
```

Comment:
Not important as this is mocked, but shouldn't we use settimeout instead as the promise will only resolve once anyway

Reply:
Ahaha yes of course it's the wrong functions ^^" I wante to use setTimeout here